matlab/fiff_write_double_complex.m

```diff
@@ -45,15 +45,21 @@ function fiff_write_double_complex(fid,kind,data)
 if count ~= 1
    error(me,'write failed');
 end
-for k = 1:nel
-   count = fwrite(fid,real(data(k)),'double');
-   if count ~= 1
-      error(me,'write failed');
-   end
-   count = fwrite(fid,imag(data(k)),'double');
-   if count ~= 1
-      error(me,'write failed');
-   end
+% this is quite slow
```

Comment:
Rather than comment this stuff out I think we should just delete it. It's in the `git` history easily browsable on GitHub (and findable with `git blame` view on GitHub) so seems cleaner to just have the new vectorized implementations

Reply:
OK, done